micro_sam/multi_dimensional_segmentation.py

```diff
@@ -29,6 +29,7 @@ def segment_mask_in_volume(
     projection: Union[str, dict],
     progress_bar: Optional[Any] = None,
     box_extension: float = 0.0,
+    verbose: bool = True
```

Comment:
I would set this to `False` by default. Otherwise we will get a lot of output in the plugin.

Reply:
Done! ;)